cogs/images.py

```diff
@@ -20,45 +20,80 @@
 from discord.ext import commands
 
 # misc imports
+import concurrent.futures
 import os
 import numpy as np
 import cv2
 import math
-from functools import wraps
+from functools import wraps, partial
```

Comment:
while you're at it, maybe add a decorator for the cpu intensive tasks :wink:

Reply:
due to various issues with how python passes functions to `run_in_executor` and pickling, i don't think this is currently possible